doc/modules/svm.rst

```diff
@@ -453,7 +453,7 @@ Tips on Practical Use
     set to ``False`` the underlying implementation of :class:`LinearSVC` is
     not random and ``random_state`` has no effect on the results.
 
-  * Using L1 penalization as provided by ``LinearSVC(loss='l2', penalty='l1',
+  * Using L1 penalization as provided by ``LinearSVC(loss='squared_hinge', penalty='l1',
```

Comment:
Since `loss='squared_hinge'` is the default, we can remove it:

```suggestion
  * Using L1 penalization as provided by ``LinearSVC(penalty='l1',
```